WordPress/Resources/en.lproj/Localizable.strings

```diff
@@ -9485,6 +9485,8 @@
 /* This text is used when the user is configuring the iOS widget to suggest them to select the site to configure the widget for */
 "ios-widget.gpCwrM" = "Select Site";
 
+/* This text is required by Apple to be part of the tranlsations but is not shown to the users. You can ignore translating this. */
+"ios-widget.tVvJ9c" = "Select Site Intent";
+
```

Comment:
I'm not convinced it was worth manually applying that change in the `Localizable.strings` file Goven that this PR targets `trunk` and this file will be regenerated from scratch (from code using genstrings + from merge of Sites.strings afterwards) during each code freeze 🙃